sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -186,39 +186,34 @@ export interface ExtractKeyPhrasesSuccessResult extends TextAnalyticsSuccessResu
 export interface HealthcareEntitiesArray extends Array<HealthcareResult> {
 }
 
-// @public (undocumented)
+// @public
 export type HealthcareEntity = Entity & {
     isNegated: boolean;
-    links?: HealthcareEntityLink[];
+    dataSource?: HealthcareEntityDataSource[];
+    relatedHealthcareEntities: Map<HealthcareEntity, HealthcareEntityRelationType>;
```

Comment:
I'm still confused about this use of a Map. Some sample code would be helpful for me to understand this design and the way we expect a customer to use this. Is there sample code already?